auto_submit/lib/requests/check_pull_request.dart

```diff
@@ -6,12 +6,14 @@ import 'dart:async';
 import 'dart:convert';
 
 import 'package:github/github.dart';
+import 'package:graphql/client.dart' hide Request, Response;
```

Comment:
See https://github.com/SpinlockLabs/github.dart/blob/master/lib/src/common/checks_service.dart#L146 for listing check runs